educational_modules/tasks.js

```diff
@@ -196,5 +226,77 @@ module.exports = [
       form: 'symptoms_check',
       label: 'task.symptoms_check.title',
     }],
+  },
+  {
+    name: 'covid_education_module_1',
+    icon: 'icon-healthcare',
+    title: 'COVID Modules: Education (1/3)',
+    appliesTo: 'contacts',
+    appliesToType: ['person'],
+    appliesIf: function (contact) {
+        return contact.contact.role === 'chw';
+    },
+    resolvedIf: function (contact) {
+        this.mostRecentCovidEducation1 = Utils.getMostRecentReport(contact.reports, 'covid_education');
```

Comment:
A best practice to follow when determining the scope of a variable is to use the narrowest scope possible. If a variable can be scoped to exist within a function, there is no value to make it a global.  I'd encourage you to think a bit about the scoping of this variable. 

`this.mostRecentCovidEducational1` declares a variable and attaches it to the `this` object (in this case, `this` = the current task object). The scope of this variable is broad because it is attached to a long-living object -- `this` is a mechanism used to scope variables so they can be reused between functions within a task which doesn't seem necessary here.

Could this be `const mostRecentCovidEducation1`?

Reply:
Returned Utils.getMostRecentReport(contact.reports, 'covid_education') instead of creating a variable.

Reply:
Looks like you're still using `this.mostRecentCovidEducation1` I think you can replace this with 

```
const mostRecentCovidEducation1 = Utils.getMostRecentReport(contact.reports, 'covid_education');
return mostRecentCovidEducation1 && isCovidEducationValid(mostRecentCovidEducation1);
```

You might also consider having `isCovidEducationValid` return false if the parameter isn't what is expected. That might make the last line a bit more readable.